### PR TITLE
Added a script to generate password, set Okta password, and send an email (IAMInitOktaUser)

### PR DESCRIPTION
**Pull request by idovandijk**
## Status
Ready

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-4558

## Description
Added a script that:
- Generates a password
- Sets the password for an Okta user
- Sends an email with the password

This script is a wrapper script used for IAM use-cases, and the goal is to allow user passwords resets without exposing the password in XSOAR.

## Minimum version of Cortex XSOAR
6.5.0

## Does it break backward compatibility?
No

### CODE REVIEW

File: Packs/Okta/Scripts/IAMInitOktaUser/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Generates a password and sets an Okta user account with that password.
```
Comment (dantavori) [inline Packs/Okta/Scripts/IAMInitOktaUser/README.md:1]: please generate a doc using the sdk command (or copy the AD one and change manually)

File: Packs/Okta/ReleaseNotes/3_1_0.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Scripts
+##### New: IAMInitOktaUser
+- Generates a password and sets an Okta user account with that password, enables the account and sends an email to the user with the account information.
```
Comment (dantavori) [inline Packs/Okta/ReleaseNotes/3_1_0.md:4]: don't forget to update here the description too (in addition to the yml as discussed)

**Review by dantavori — APPROVED** [review]
nice! see comments and feel free to merge when they're resolved

File: Tests/conf.json [hunk 3]
```diff
@@ -4988,6 +4988,11 @@
             "fromversion": "6.0.0",
             "memory_threshold": 90
         },
+        {
+            "playbookID": "IAMInitOktaUser - Test",
+            "integrations": ["Okta v2","EWSO365"],
+            "fromversion": "6.5.0"
+        },
```
Comment (ostolero) [inline Tests/conf.json:4995]:
```suggestion
                    "instance_names": ["{enter_instance_name}"]
        },
```
Test fails in the build, choose the right instance to run EWS0365 with 
```
[2022-12-19 15:25:00] - [MainThread] - [ERROR] - There are 2 instances of EWSO365, please select one of them by using the instance_name argument in conf.json. The options are:
ewso365_dev_team
ewso365_sec_eng_team
```

**Pull request merged**